src/dc_mimefactory.rs

```diff
@@ -1206,11 +1200,9 @@ unsafe fn build_body_file(
                 .format(&format!("voice-message_%Y-%m-%d_%H-%M-%S.{}", suffix))
                 .to_string();
             filename_to_send = to_cstring(res);
-        } else if (*msg).type_0 == DC_MSG_AUDIO as libc::c_int {
+        } else if (*msg).type_0 == DC_MSG_AUDIO {
             filename_to_send = dc_get_filename(pathNfilename)
-        } else if (*msg).type_0 == DC_MSG_IMAGE as libc::c_int
-            || (*msg).type_0 == DC_MSG_GIF as libc::c_int
-        {
+        } else if (*msg).type_0 == DC_MSG_IMAGE || (*msg).type_0 == DC_MSG_GIF as libc::c_int {
```

Comment:
Maybe remove the ending `as libc::c_int`?

Reply:
done